Kconfig

```diff
@@ -140,4 +140,27 @@ config BUILD_VM_ROM
 	help
 	  Select if you want to build VM ROM
 
+menu "Performance measurement"
+
+config PERF_MEASURE
+	bool "Enable performance measurement"
+	default n
+	help
+	  Select for performance measurment of various FW components.
```

Comment:
What does it mean various FW components? The phrasing is vague. Maybe remove various.